stern/tail.go

```diff
@@ -171,6 +174,18 @@ func (t *Tail) Start(ctx context.Context) error {
 	return err
 }
 
+func (t *Tail) Resume(ctx context.Context, resumeRequest *ResumeRequest) error {
+	sinceTime, err := resumeRequest.sinceTime()
+	if err != nil {
+		fmt.Fprintf(t.errOut, "failed to resume: %s, fallback to Start()\n", err)
+		return t.Start(ctx)
+	}
+	t.resumeRequest = resumeRequest
+	t.Options.SinceTime = sinceTime
+	t.Options.SinceSeconds = nil
+	return t.Start(ctx)
```

Comment:
```suggestion
	t.Options.SinceTime = sinceTime
	t.Options.SinceSeconds = nil
	t.Options.TailLines = nil
	return t.Start(ctx)
```

It may be necessary to stop specifying the tail lines when resuming.

```
$ dist/stern . --timestamps --tail=1
+ counter › c
counter c 2023-02-18T10:00:55.565171697+09:00 line 705
counter c 2023-02-18T10:01:05.568947073+09:00 line 706
counter c 2023-02-18T10:01:05.569034109+09:00 line 707
counter c 2023-02-18T10:01:05.569051509+09:00 line 708
counter c 2023-02-18T10:01:05.569062139+09:00 line 709
counter c 2023-02-18T10:01:05.569072222+09:00 line 710
- counter › c
failed to tail: unexpected EOF, will retry
+ counter › c
- counter › c
failed to tail: Get "https://127.0.0.1:43169/api/v1/namespaces/default/pods/counter/log?container=c&follow=true&sinceTime=2023-02-18T01%3A01%3A05Z&tailLines=1&timestamps=true": read tcp 127.0.0.1:39074->127.0.0.1:43169: read: connection reset by peer, will retry
+ counter › c
- counter › c
failed to tail: Get "https://127.0.0.1:43169/api/v1/namespaces/default/pods/counter/log?container=c&follow=true&sinceTime=2023-02-18T01%3A01%3A05Z&tailLines=1&timestamps=true": EOF, will retry
+ counter › c
counter c 2023-02-18T10:06:15.687386785+09:00 line 865
counter c 2023-02-18T10:06:25.690838467+09:00 line 866
counter c 2023-02-18T10:06:25.690934378+09:00 line 867
counter c 2023-02-18T10:06:25.690955851+09:00 line 868
```

Reply:
Thanks, nice catch!

Reply:
I confirmed it was fixed after applying your change. 🛠️ 


```
$ dist/stern . --timestamps --tail=1
+ counter › c
counter c 2023-02-18T11:15:53.722286700+09:00 line 15
counter c 2023-02-18T11:16:03.724012300+09:00 line 16
counter c 2023-02-18T11:16:03.724066100+09:00 line 17
counter c 2023-02-18T11:16:03.724083500+09:00 line 18
counter c 2023-02-18T11:16:03.724093600+09:00 line 19
counter c 2023-02-18T11:16:03.724108800+09:00 line 20
- counter › c
failed to tail: unexpected EOF, will retry
+ counter › c
- counter › c
failed to tail: Get "https://127.0.0.1:64321/api/v1/namespaces/default/pods/counter/log?container=c&follow=true&sinceTime=2023-02-18T02%3A16%3A03Z&timestamps=true": EOF, will retry
+ counter › c
counter c 2023-02-18T11:16:13.727169100+09:00 line 21
counter c 2023-02-18T11:16:13.727198900+09:00 line 22
counter c 2023-02-18T11:16:13.727209500+09:00 line 23
counter c 2023-02-18T11:16:13.727257900+09:00 line 24
counter c 2023-02-18T11:16:13.727271700+09:00 line 25
```